mistralrs-server/src/main.rs

```diff
@@ -230,8 +230,6 @@ fn parse_request(
     let stop_toks = match oairequest.stop_seqs {
         Some(StopTokens::Multi(m)) => Some(InternalStopTokens::Seqs(m)),
         Some(StopTokens::Single(s)) => Some(InternalStopTokens::Seqs(vec![s])),
-        Some(StopTokens::MultiId(m)) => Some(InternalStopTokens::Ids(m)),
```

Comment:
This felt like a footgun to me.
Since there is no discriminator, if one passes a number as stop sequence it might be interpreted as token.

There is no unambiguous way of passing a number as stop sequence, so I removed this from open ai api.

Reply:
Thanks, you are absolutely correct!